tasmota/tasmota_template_legacy.h

```diff
@@ -420,6 +421,7 @@ const uint16_t kGpioConvert[] PROGMEM = {
   AGPIO(GPIO_OLED_RESET),     // OLED Display Reset
   AGPIO(GPIO_SOLAXX1_TX),     // Solax Inverter tx pin
   AGPIO(GPIO_SOLAXX1_RX),     // Solax Inverter rx pin
+  AGPIO(GPIO_SOLAXX1_RTS),    // Solax Inverter RTS pin
```

Comment:
Same here, please move to the end of the list.

Reply:
Same here. Not supported